client/src/document/ingredients/browser-compatibility-table/headers.tsx

```diff
@@ -60,7 +60,21 @@ function BrowserHeaders({ browsers }: { browsers: BCD.BrowserName[] }) {
 
 export function browserToIconName(browser: string) {
   const browserStart = browser.split("_")[0];
-  return browserStart === "firefox" ? "simple-firefox" : browserStart;
+  const browserEnd = browser.split("_")[1];
+  let browserIconName = browserStart;
```

Comment:
Let's simplify the whole function, while we're here:

```tsx
export function browserToIconName(browser: string) {
  if (browser.startsWith("firefox")) {
    return "simple-firefox";
  } else if (browser === 'webview_android') {
    return "webview";
  } else if (browser === 'webview_ios') {
    return "safari";
  } else {
    const browserStart = browser.split("_")[0];
    return browserStart;
  }
}
```

Reply:
It's also possible and more concise. My solution was more for case of future expansions, but that also work. I'll use yours for conciseness.